src/block/VanillaBlocks.php

```diff
@@ -1098,6 +1099,7 @@ protected static function setup() : void{
 		self::register("stone_pressure_plate", $factory->get(70, 0));
 		self::register("stone_slab", $factory->get(421, 2));
 		self::register("stone_stairs", $factory->get(435, 0));
+		self::register("stonecutter_block", $factory->get(452, 0));
```

Comment:
`_block` is redundant. We know it's a block already or we wouldn't be using `VanillaBlocks`.